bom/deployment/pom.xml

```diff
@@ -612,6 +612,11 @@
                 <artifactId>quarkus-elytron-security-jdbc-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-security-jpa-deployment</artifactId>
```

Comment:
I think the name here is incorrect. At least it's inconsistent with the other `elytron-security` extensions (they have elytron in the name).

Reply:
So this is a bit of a weird one, this is not really an Elytron extension even though it does use some of the Elytron password functionality, it does not use the Elytron realm mechanism.

Reply:
It's only using Elytron under the hood for bCrypt, which is an implementation detail.

Reply:
@stuartwdouglas apparently you were the one asking for a rename. I'm just a bit concerned that all the other security modules have Elytron in the name and I'm not really sure they depend more on Elytron than this one but maybe I'm mistaken? Not being consistent is an issue IMHO. I wasn't really fond of having Elytron in the name to begin with but I'm not sure it's a good idea to not be consistent.

Reply:
OIDC does not have Elytron in the name, as it does not use Elytron. Elytron is not the core of our security infrastructure, we have some modules that use it because we did not have the time or resources to implement some of that stuff ourselves, but Elytron is inherently blocking.